src/components/Invoice.tsx

```diff
@@ -8,108 +8,171 @@ import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import IconButton from '@mui/material/IconButton';
+import { SubmitHandler, useForm, Controller } from 'react-hook-form';
+
+
+interface IPersonalData {
+
```

Comment:
Is prettier working correctly here? Formatting of this file is pretty strange.

Reply:
outdated